.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
         - rustup component add clippy
       script:
         - cd driver/
-        - cargo clippy -- -D warnings
+        - cargo clippy -- cap-lints  # -- clippy has an issue: https://github.com/rust-lang/rust-clippy/issues/3874
```

Comment:
I saw that one of the suggestions in the thread here was to use `--cap-lints`. Could you mentioned a little more about what it does?

Reply:
I could really not find any good documentation of that.

If Clippy is linting all the code, it also needs to know the code of the dependencies, as it needs to infer some types. That is why Clippy is looking into the mongodb library, but in there the clippy.toml file is not valid with the new rust version. This is causing the issue. Cap-lints is supposed to cut off the linting so that especially we no longer have to compile the libraries. 

But obviously, it is not working and I can not find a solution right now.

Reply:
They pushed a fix to the mongodb repo yesterday: https://github.com/mongodb-labs/mongo-rust-driver-prototype/commit/9203ec3216976cfbfddb07856b25d061965810af

Pinning the dependency to that commit should fix the issue.

Reply:
NICE!

Thanks a lot for unblocking